output/mistral_elasticsearch/mistral_elasticsearch.c

```diff
@@ -590,6 +596,8 @@ void mistral_startup(mistral_plugin *plugin, int argc, char *argv[])
  */
 void mistral_exit(void)
 {
+    LOG(plugin, MAJOR, "Received mistral_exit message");
```

Comment:
No, this is called just as the plugin is about to exit (only in the successful case, interestingly).